src/test/java/com/google/api/generator/gapic/composer/ComposerTest.java

```diff
@@ -14,22 +14,17 @@
 
 package com.google.api.generator.gapic.composer;
 
-import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.*;
```

Comment:
Can you please update Intellij to prevent it from importing *?

Reply:
Thanks - I've updated my IntelliJ settings so this shouldn't happen anymore.